### PR TITLE
removed bears/tests

### PR DESCRIPTION
**Pull request by BigDproject**
solved issue  #1821

### CODE REVIEW

**Comment by mr-karan** [comment]
unack https://github.com/coala-analyzer/coala/pull/1986/commits/69fa95d1955e8f929442f1a9c7c76eb0207aa4e0

File: docs/Getting_Involved/Writing_Tests.rst [hunk 1]
```diff
@@ -177,8 +177,7 @@ Kickstart
 This section contains a concluding and simple example that you can use
 as a kickstart for test-writing.
 
-Put the code under the desired folder inside ``tests`` or
-``bears/tests``, modify it to let it test your stuff and run the test from
+Put the code under the desired folder inside ``tests``, modify it to let it test your stuff and run the test from
```
Comment (mr-karan) [inline docs/Getting_Involved/Writing_Tests.rst]: Doesn't follow PEP8. Run coala-ci over your code.

**Comment by AbdealiJK** [comment]
Hey there :)

As you can see, gitmate detected a problem. Please run coala on your code base before submitting it to avoid these.

In the commit message use a capital D in "delete". Probably "Remove" is a better verb ? Although Delete is fine too I guess :P
otherwise looks good :+1:

File: docs/Getting_Involved/Writing_Tests.rst [hunk 2]
```diff
@@ -177,8 +177,8 @@ Kickstart
 This section contains a concluding and simple example that you can use
 as a kickstart for test-writing.
 
-Put the code under the desired folder inside ``tests`` or
-``bears/tests``, modify it to let it test your stuff and run the test from
+Put the code under the desired folder inside ``tests``,
```
Comment (tushar-rishav) [inline docs/Getting_Involved/Writing_Tests.rst]: running `coala` would fix these bugs.  :D

**Comment by BigDproject** [comment]
Thankn you for your advice. I will try to do my best and do another commit

**Comment by sils1297** [comment]
unack https://github.com/coala-analyzer/coala/pull/1986/commits/7d2e7e7004f830eda4cc54cb2eb5c3a9263429f3

**Comment by SanketDG** [comment]
ack c210ecc

**Comment by sils1297** [comment]
see https://github.com/coala-analyzer/coala/pull/1989

**Pull request closed without merging**